hcaptcha/build.gradle

```diff
@@ -0,0 +1,106 @@
+import groovy.text.SimpleTemplateEngine
+import java.util.stream.Collectors
+
+apply from: configs.androidLibrary
+
+apply plugin: 'checkstyle'
+apply plugin: "org.jetbrains.kotlin.plugin.parcelize"
+apply plugin: 'org.jetbrains.kotlin.plugin.serialization'
+
+dependencies {
+    implementation libs.androidx.appCompat
+    implementation libs.kotlin.serialization
+
+    testImplementation testLibs.junit
+    testImplementation testLibs.kotlin.test
+    testImplementation testLibs.mockito.inline
+    testImplementation testLibs.mockito.kotlin
+}
+
+android {
+    compileSdk 34
```

Comment:
A lot of the properties in the android block here aren't wanted or needed.

Reply:
great catch. I cleaned a lot of this up, lmk what you think.